pyscriptjs/tests/py-unit/test_python_plugins.py

```diff
@@ -130,7 +132,7 @@ def test_connected_calls(self, plugins_manager: ppt.PluginsManager):
         # - init the plugin instance passing the plugins_manager as parent app
         # - add the plugin instance to plugins_manager.plugins
         assert not py_tutor.plugin.app
-        plugins_manager.addPythonPlugin(py_tutor.plugin)
+        await plugins_manager.addPythonPlugin(py_tutor.plugin)
 
         # EXPECT: the plugin app to now be the plugin manager
         assert py_tutor.plugin.app == plugins_manager
```

Comment:
How was this passing before? Did you actually test the failure mode and the it passes after you fixed the issue?